connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -1216,6 +1214,9 @@ class SparkConnectPlanner(val session: SparkSession) {
       case "unwrap_udt" if fun.getArgumentsCount == 1 =>
         Some(UnwrapUDT(transformExpression(fun.getArguments(0))))
 
```

Comment:
would you mind move it just above the `// ML-specific functions` and add a comment like `// PS(Pandas API on Spark)-specific functions`, since I guess you will add a group of functions for PS :)